.github/workflows/tox.yml

```diff
@@ -0,0 +1,21 @@
+name: tox
+
+on: pull_request
+
+jobs:
+  lint:
```

Comment:
Need to rename this to something else besides lint as the flake8 test also uses lint as the job identifier. I think `unit` is fine imo.